recipes/coiled-runtime/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "coiled-runtime" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/coiled/coiled-runtime/dist/conda/noarch/{{ version }}/coiled-runtime-{{ version }}.tar.gz
```

Comment:
This URL doesn't appear to exist. I think we want to issue a `0.0.3` release tag over in https://github.com/coiled/coiled-runtime and then update this to point to something like `https://github.com/coiled/{{ name }}/archive/{{ version }}.tar.gz`